src/view/Home/components/Footer/style.ts

```diff
@@ -15,7 +15,11 @@ export const FooterMedias = styled.div`
 export const AnchorCask = styled.a`
   cursor: pointer;
 
+  :nth-child(1) {
+    margin: 0 8px;
+  }
+
```

Comment:
Mestre, oq acha de remover esse `nth-child` já que todos os elementos vão ter esse margin?

```suggestion
    margin: 0 8px;
 

```